homeassistant/components/simplefin/strings.json

```
@@ -6,6 +6,13 @@
         "data": {
           "api_token": "Claim Token or Access URL"
         }
+      },
+      "reauth_confirm": {
+        "title": "SimpleFin - Re-authentication Required",
```

Use existing translations?

```suggestion
        "title": "[%key:common::config_flow::title::reauth%]",
```